src/test/java/Hmwk18.java

```diff
@@ -37,9 +38,14 @@ public void AddSongToPlaylist() throws InterruptedException {
 
     }
 
+    public void ValidateSongIsPlaying(){
+        WebElement ValidateSongPlaying = driver.findElement(By.xpath("//button[@title='Click for a marvelous visualizer']"));
+        ValidateSongPlaying.click();
+        Assert.assertTrue(ValidateSongPlaying.isDisplayed());
+    }
     private void PlaySong() {
 
-        WebElement PlaySong = driver.findElement(By.xpath("//ul/li['playback]"));
+        WebElement PlaySong = driver.findElement(By.xpath("//span[data-testid='play-btn']"));
```

Comment:
Try like this:

 ```
private void PlaySong() {
        WebElement PlaySong = driver.findElement(By.xpath("//span[@data-testid='play-btn']"));
        Actions actions = new Actions(driver);
        actions.click(PlaySong).perform();

    }

```

![image](https://user-images.githubusercontent.com/104761482/199681142-bb7ce883-58f8-4136-bead-2a94c6322e37.png)